.github/workflows/enos-run.yml

```diff
@@ -46,7 +46,7 @@ on:
         default: '"ubuntu-latest"'
       ssh-key-name:
         type: string
-        default: enos-ci-ssh-key
+        default: vault-ci-ssh-key
```

Comment:
since this workflow will only be run in ci, we can set the default at `${{ github.event.repository.name }}-ci-ssh-key` too ?

Reply:
@jaymalasinha Yeah that's a good suggestion. I'll give it a try

Reply:
Done.